store/localstore/mvcc_test.go

```diff
@@ -173,6 +173,28 @@ func (t *testMvccSuite) TestSnapshotGet(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (t *testMvccSuite) TestMvccSuiteGetLatest(c *C) {
```

Comment:
As I understand, this pr is used for multiple txns runing simultaneously. For example:
txn_0.begin()
txn_0.set(k, v0)
txn_0.commit()
txn_1.begin()
txn_2.begin()
txn_2.set(k, v1)
txn_2.commit()
v := txn_1.get(k)
Assrt(v, Euqals, v0)

If I am right, I do not see the unit test simulates this scenario.